api/v01/entities/vrp_result.rb

```diff
@@ -63,9 +66,13 @@ class VrpResultSolutionRouteActivities < Grape::Entity
       expose :begin_time, documentation: { type: Integer, desc: 'Time visit starts' }
       expose :end_time, documentation: { type: Integer, desc: 'Time visit ends' }
       expose :departure_time, documentation: { type: Integer, desc: '' }
-      expose :service_id, documentation: { type: String, desc: 'Internal reference of the service' }
-      expose :pickup_shipment_id, documentation: { type: String, desc: 'Internal reference of the shipment' }
-      expose :delivery_shipment_id, documentation: { type: String, desc: 'Internal reference of the shipment' }
+      expose :service_id, expose_nil: false, documentation: { type: String, desc: 'Internal reference of the service' }
+      # expose :original_service_id, expose_nil: false, documentation: { type: String, desc: 'Original reference of the service' } # TODO : rename to service_id and output when internal service id renamed (https://gitlab.com/mapotempo/optimizer-api/-/issues/537)
```

Comment:
The next change of the API should be the v2. Which will merge all 'id' fields into a single one and one `original_id`

We should only mention `original_id` instead of o_service_id, o_shipment_id, o_rest_id and o_vehicle_id

Reply:
We can even do just `id`, no?